src/opm/output/eclipse/Summary.cpp

```diff
@@ -1891,6 +1891,9 @@ void updateValue(const Opm::EclIO::SummaryNode& node, const double value, Opm::S
     else if (node.category == Opm::EclIO::SummaryNode::Category::Group)
         st.update_group_var(node.wgname, node.keyword, value);
 
+    else if (node.category == Opm::EclIO::SummaryNode::Category::Network)
```

Comment:
The `Category` here is [`Node`](https://github.com/OPM/opm-common/blob/a43d9245e875f2a6998c7eacb48077366fa13d6b/opm/io/eclipse/SummaryNode.hpp#L40).  As an aside, is there a reason for not fusing this branch into the previous (`Group`) condition since they both execute the same statement?

Reply:
Thanks; no real reason - it stands out more with the current approach.